index.bs

```diff
@@ -958,6 +959,8 @@ The <dfn>device type</dfn> indicates the kind of device used for the context. It
 <dd>Provides the broadest compatibility and usability across all client devices with varying degrees of performance.</dd>
 <dt>"<dfn enum-value>gpu</dfn>"</dt>
 <dd>Provides the broadest range of achievable performance across graphics hardware platforms from consumer devices to professional workstations.</dd>
+<dt>"<dfn enum-value>npu</dfn>"</dt>
+<dd>Provides narrower functionality on dedicated machine learning devices which perform between CPU and GPU, favoring other factors over raw performance like lower power usage for longer battery life or lower latency for results. The backend implementation may fall back to other devices for certain operators and parts of the graph.</dd>
```

Comment:
I don't quite get the meaning of "perform between CPU and GPU". Does it refer to its power and performance characteristic?

Reply:
Here's a proposal that avoids mentioning CPU and GPU. Feel free to use it, or parts of it, if this sounds reasonable:

"Provides power efficiency for sustained workloads across hardware platforms with purpose-built accelerators. The backend implementation may fall back to other devices for certain operators and parts of the graph."

My expectation is that this descriptive text will receive updates in the future when more products of this emerging category become available. That is to say, I'd expect us to put in here text that reflects the group's current best understanding.

Reply:
> may fall back to other devices

I think the problem was not the mentioning the CPU and GPU, but the expression of what happens ("perfom between").
The `other devices` might be misleading, should we rather use the same words (in your suggested text, "other accelerators"), eventually "other accelerators, e.g. CPU or GPU"?

Reply:
> I don't quite get the meaning of "perform between CPU and GPU". Does it refer to its power and performance characteristic?

**Ningxin**: Yep, I meant the execution performs faster than CPU and slower than GPU (but moot point now since I used Anssi's suggested text).

> Here's a proposal that avoids mentioning CPU and GPU. Feel free to use it, or parts of it, if this sounds reasonable

**Anssi**:👍 Used as-is.

> should we rather use the same words ...  "other accelerators, e.g. CPU or GPU"?

**Zoltan**: The CPU isn't thought of as an "accelerator", and using the word "device" is consistent with the enum name `MLDeviceType`. So I'd keep it as device.